src/github/client/github-queries.ts

```diff
@@ -361,12 +362,50 @@ export type getDeploymentsResponse = {
 	}
 };
 
+export const getDeploymentsQueryByCreatedAtDesc = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
+  repository(owner: $owner, name: $repo){
+    deployments(first: $per_page, after: $cursor, orderBy: { direction: DESC, field: CREATED_AT }) {
+      edges {
+        cursor
+        node {
+					createdAt
+          repository {
+            id: databaseId
+            node_id: id
+            name
+            owner {
+              login
+            }
+          }
+          databaseId
+          commitOid
+          task
+          ref {
+            name
+            id
+          }
+          environment
+          description
+          latestStatus {
+            environmentUrl
+            logUrl
+            state
+            id
+            updatedAt
+          }
+        }
+      }
+    }
+  }
+}`;
+
 export const getDeploymentsQuery = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
   repository(owner: $owner, name: $repo){
     deployments(first: $per_page, after: $cursor) {
       edges {
         cursor
         node {
+					createdAt
```

Comment:
Not sure I need to have this, as it is to match the typing. Otherwise typing can be `?`?